src/Managed/GoogleAnalytics.Core/project.json

```diff
@@ -2,10 +2,7 @@
   "supports": {},
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.1",
-    "NETStandard.Library": {
-      "version": "1.6.0",
-      "suppressParent": "all"
-    }
+    "NETStandard.Library": "2.0.2"
```

Comment:
Please don't bump to .NET Standard 2.0, then mobile won't be supported anymore (and there is no real reason to do so)

Reply:
More importantly, this should move to use ProjectReference.